CMU-17-356/cmu-17-356-final-project-s23-walker
# [PR] Creating Walker Call

## Description
Resolves #37 
Resolves #51 
Resolves #52 
* Completes functionality of create walker call
* Adds in time picker to page, however datatype still stored as Date in database as it includes times
* Fixed problem of calls on the same day being displayed as invalid due to validation
* Clearing null entries and adding error handling
* Adding navigation back to home page after creating the walker call
* Changing to radio buttons to ensure that the choice is always either, not neither or none as that would be a wrong call

## How Has This Been Tested?
* Make sure backend is running (`cd walker-backend`, `npm install`, `npm run dev`)
* Make sure frontend is running (`cd walker`, `npm install`, `npm start`)
* Created call via the "Create Walker Call" option
* Checked on MongoDB that the call went though, went back to home page to ensure it was displayed properly
* Checked various times and dates

## Types of changes




## Checklist:



-   [ ] My change requires a change to the documentation, which is located at the [Wiki](https://github.com/CMU-17-356/cmu-17-356-final-project-s23-walker/wiki)

## Other issues / help required
